.env.example

```
@@ -74,7 +74,27 @@ CELERY_ENABLE_UTC=true
 # India Standard Time (UTC+05:30)
 CELERY_TIMEZONE=Asia/Kolkata
 
+# Flower Configuration (Celery Monitoring)
+FLOWER_PORT=5555
+FLOWER_BASIC_AUTH=admin:changethis
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FLOWER_BASIC_AUTH key should go before the FLOWER_PORT key

(UnorderedKey)